recipes/mapbox-wagyu/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get, apply_conandata_patches, export_conandata_patches
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.52.0"
+
+class MapboxWagyuConan(ConanFile):
+    name = "mapbox-wagyu"
+    description = "A general library for geometry operations of union, intersections, difference, and xor"
+    license = "LicenseRef-mapbox-wagyu"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mapbox/wagyu/"
+    topics = ("geometry", "clipping", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
```

Comment:
your adding `__has_include` make sure the minimum compiler versions reflect the support for this. See  https://en.cppreference.com/w/c/compiler_support